lighthouse-core/lib/page-functions.js

```diff
@@ -406,6 +405,23 @@ function getBoundingClientRect(element) {
   };
 }
 
+/**
+ * @param {HTMLElement} element
+ */
+/* instanbul ignore next */
+function getImageSource(element) {
```

Comment:
it's a little more complicated unfortunately  :/ https://github.com/GoogleChrome/lighthouse/blob/9cf4511e073be3c6518ed8693ce43da96f41a5a7/lighthouse-core/gather/gatherers/image-elements.js#L63-L66

can we use the data from `ImageElements` artifact and find it using the devtools node path data or something instead of duplicating in `TraceElements`?

Reply:
For sure. Gonna look into this.